internal/jobservice/application.go

```diff
@@ -41,19 +42,26 @@ func (a *App) StartUp(ctx context.Context) error {
 	inMemoryMap := make(map[string]*repository.JobTable)
 	subscribedJobSets := make(map[string]*repository.SubscribeTable)
 	jobStatusMap := repository.NewJobStatus(inMemoryMap, subscribedJobSets)
-	inMemoryJobService := repository.NewInMemoryJobServiceRepository(jobStatusMap, config)
-	jobService := server.NewJobService(config, *inMemoryJobService)
+
+	db, err := sql.Open("sqlite", "jobservice.db")
```

Comment:
this path should likely be configurable

Reply:
I agree with you but this is the name of the database for this library.  It is not the path.  This library only takes what you pass as second parameter to be the name of the database.  Unclear of ways around that.  

Now I can make the name of the databse configurable.

Reply:
What needs to be configurable is the place on disk the jobservice.db is created/opened from. That may include being able to change the name.